recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -49,35 +82,88 @@ def configure(self):
             except Exception:
                 pass
 
+        if not self.options.with_otlp:
+            try:
+                del self.options.with_otlp_grpc
```

Comment:
This should use `rm_safe` :)

Reply:
required_conan_version should be 1.53 too

Reply:
Ah I'm glad we now have a first-class way of doing this instead of the try/del/except!